MTA-7229 [DOC] Configuring repositories documentation is misleading (#384)
migtools/mta-documentation: docs/topics/mta-ui/proc_configuring-git-repos.adoc

:_mod-docs-content-type: PROCEDURE
[id="configuring-git-repos_{context}"]
= Enable insecure access to Git repositories

[role="_abstract"]

> Administrators can configure {ProductFullName} to accept self-signed or internal certificates to allow an insecure connection to access the *Git* repository internal to your organization.

Reviewer: We must use this certain structure for procedures in terms of short descriptions:
```suggestion
As an Administrator, configure {ProductFullName} to accept self-signed or internal certificates to allow an insecure connection to your internal *Git* repository.
```
